pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -533,7 +533,7 @@ func (c *DisruptionBudgetController) sync(key string, vmiExists bool, vmi *virtv
 				return c.shrinkPDB(vmi, pdb)
 			}
 		}
-	} else if migratableOnDrain {
+	} else if needsEvictionProtection {
 		// pdb doesn't exist, create if vmi is set to migrate during drain.
```

Comment:
can you change this comment too since it just refers to migration?

Reply:
yup, fixed